### ISSUE TITLE
Neutrale Wertung für Zitate?

### ISSUE THREAD

**Issue by auge8472** [ckruse/cforum#561]
Mir fehlt für zur Abstimmung stehende Zitate die früher mögliche neutrale Wertung. Es gibt Vorschläge, die kann ich nicht positiv oder negativ bewerten und es gibt welche, da möchte ich es nicht.

So wurde ein Satz von mir als Zitatvorschlag eingereicht. Lehne ich das Zitat ab, ist es komisch. Goutiere ich das Zitat, ist es noch komischer. Lasse ich es unbewertet stehen, schaue ich immer wieder nach, welches von mir unbemerkte Zitat da unbewertet in der Gegend herumsteht.

Gibt es einen Grund, der dazu führte, dass diese Bewertungsmöglichkeit wegfiel oder wurde das bei der Neuimplementierung vergessen?

**Comment by campingrider** [comment]
>  Lasse ich es unbewertet stehen, schaue ich immer wieder nach, welches von mir unbemerkte Zitat da unbewertet in der Gegend herumsteht.

Das Problem ist mMn nicht so sehr die Möglichkeit zur neutralen Bewertung (die halte ich nicht für dringend notwendig), sondern, dass sich die "1" in den Klammern hinter "Zitatesammlung" nicht sinnvoll verhält. Im Moment zeigt die an "es gibt x unbewertete Zitatvorschläge", besser wäre aber: "es gibt x bisher ungelesene Zitatvorschläge".

Frage an Christian: Ist es technisch machbar, den Counter hinter "Zitatesammlung" so zu gestalten, dass er sich an den bisher ungelesenen Zitat-Benachrichtigungen orientiert? Wobei auch das problematisch ist - jemand wie der Martin, der keine Benachrichtigungen nutzt, würde dann nie sehen, dass der Counter herunterzählt (weil die Benachrichtigung nur dann ungelesen wird, wenn man auch auf die Benachrichtigung gedrückt hat, um zum Zitatvorschlag zu kommen).

Vielleicht ist es im Lichte dieser Probleme nötig, den Counter entweder so einzustellen, dass er ein eigenes gelesen-System benutzt (vergleichbar mit den Zahlen, die im title angezeigt werden - die letzte davon richtet sich auch zielsicher an den ungelesenen Beiträgen) oder, falls das zu aufwändig ist, den Counter eher wieder zu entfernen - seit es die Benachrichtigungen für ungelesene Zitate gibt, ist er mMn nicht mehr so nötig wie davor.

**Comment by MatthiasApsel** [comment]
s.a. https://forum.selfhtml.org/meta/2015/dec/9/benachrichtigung-bei-neuem-zitat-vorschlag/1656824#m1656824 und drumrum

**Comment by auge8472** [comment]
> Das Problem ist mMn nicht so sehr die Möglichkeit zur neutralen Bewertung (die halte ich nicht für dringend notwendig), sondern, dass sich die "1" in den Klammern hinter "Zitatesammlung" nicht sinnvoll verhält. Im Moment zeigt die an "es gibt x unbewertete Zitatvorschläge", besser wäre aber: "es gibt x bisher ungelesene Zitatvorschläge".

Kann man so sehen, muss man aber nicht. Ich halte es schon für richtig, dass angezeigt wird, dass X zu bewertende Zitate *noch nicht bewertet* wurden. Klar, mit der von dir bevorzugten Lösung lassen sich zur Wahl stehende Zitate besser ignorieren. Einmal die Seite augerufen und die Zahl springt bis zum nächten Zitatvorschlag wieder auf 0 zurück. Das Gefühl, etwas vergessen zu haben, beschleicht mich dann also nur noch dann, wenn ich die Seite wegen des nächsten Vorschlags wieder aufrufe. Damit ließe sich mein letzter Punkt verbergen, aber eben nur verbergen.

Aufgelistet werden die unbewerteten Vorschläge aber, völlig zu recht, dennoch. Ich will sie ja auch bewerten, aber zu manchen Zitatvorschlag will ich weder *ja* noch *nein* sagen und dennoch abstimmen (nicht nur, weil ich die Liste bereinigt haben will). Genau an dieser Stelle halte ich die Möglichkeit der neutralen Bewertung für so wichtig, dass ich nach dem Warum des Weglassens eines früher vorhandenen Features und der Möglichkeit für dessen Reïmplementierung frage.

**Comment by JuliusPC** [comment]
> Im Moment zeigt die an "es gibt x unbewertete Zitatvorschläge", besser wäre aber: "es gibt x bisher ungelesene Zitatvorschläge".

Finde ich gut, einige Zitate möchte ich nicht bewerten, aber sie auch nicht mehr „vorgesetzt bekommen“

**Comment by MatthiasApsel** [comment]
Du kannst es bewerten und die Bewertung zurücknehmen.

**Comment by JuliusPC** [comment]
Funktioniert bei mir leider nicht. Auch nicht mit Neuladen zwischen drin...

**Comment by auge8472** [comment]
> Du kannst es bewerten und die Bewertung zurücknehmen.

Das ist aber auch nur ein drumherum arbeiten (wenn es denn funktioniert). Damit könnte ich in Hinsicht auf das Zurücksetzen des Zählers leben. Das ungute Gefühl, ein eigenes Zitat nicht bewerten zu können, ohne es goutieren oder ablehnen zu müssen – womit dieses Issue seinen Anfang nahm –, blebt aber. Sauberer wäre daher die Möglichkeit der neutralen Bewertung.

Die Frage, warum dieses Feature *entfernt* wurde, bleibt aber unbeantwortet.

**Comment by ckruse** [comment]
Das Feature wurde nicht entfernt, das Feature gab es nie.

**Comment by JuliusPC** [comment]
> Das Feature wurde nicht entfernt, das Feature gab es nie.

Ich denke, er bezog sich auf CForum 3.x bzw. die externe Zitatsammlung.

**Comment by MatthiasApsel** [comment]
Die Zahl bleibt solange, bis du die entsprechende Benachrichtigung klickst. Deine Bewertung kannst du jederzeit ändern, auch zurücknehmen.

**Comment by ckruse** [comment]
Nein, das stimmt nicht. Die Zahl bleibt, bis man bewertet hat und taucht 
wieder auf, wenn die Bewertung zurück genommen wird.

**Comment by auge8472** [comment]
> > > Die Frage, warum dieses Feature entfernt wurde, bleibt aber unbeantwortet.
> > 
> > Das Feature wurde nicht entfernt, das Feature gab es nie.
> 
> Ich denke, er bezog sich auf CForum 3.x bzw. die externe Zitatsammlung.

Genau *das* tat er. Dass die Zitatesammlung nicht zur Forensoftware gehörte, habe ich dabei allerdings übersehen.
